nntrainer/meson.build

```diff
@@ -43,6 +43,7 @@ endforeach
 
 nntrainer_common_sources = [
   'nntrainer_logger.cpp',
+  'app_context.cpp'
```

Comment:
app_context.h must be added to nntrainer_headers if you it want to be able to import in applications.

Reply:
Done. Thanks